docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-107.adoc

```diff
@@ -0,0 +1,44 @@
+== OpenAI API Key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|TBD
+
+|Checkov Check ID
+|CKV_SECRET_107
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+An OpenAI API Key is a secure and unique identifier used to authenticate requests to OpenAI's API services, including access to powerful AI models for various applications. The API key ensures that only authorized users or applications can interact with OpenAI's services, facilitating the control and management of usage according to the specified terms and conditions. Exposed API keys can lead to exposed data and unplanned charges.
+
+=== Fix - Buildtime
+
+*OpenAI*
+
+To address the exposure of an OpenAI API Key, follow these remediation steps to ensure that your access remains secure. OpenAI provides a mechanism to manage API keys directly through its platform:
```

Comment:
If your OpenAI API key is exposed, immediately regenerate a new key through the OpenAI platform and update all applications using it to prevent unauthorized access.